flink-formats/flink-avro/src/main/java/org/apache/flink/formats/avro/AvroRowDataDeserializationSchema.java

```diff
@@ -168,7 +168,7 @@ public int hashCode() {
 
 	// --------------------------------------------------------------------------------------------
 
-	private RowData convertAvroRecordToRowData(Schema schema, RowType rowType, IndexedRecord record) {
+	public static GenericRowData convertAvroRecordToRowData(Schema schema, RowType rowType, IndexedRecord record) {
```

Comment:
package privilege is enough here
```suggestion
	static GenericRowData convertAvroRecordToRowData(Schema schema, RowType rowType, IndexedRecord record) {
```